src/parser/generated_parser.ts

```diff
@@ -18,7 +18,8 @@ import {
 import { pyNone, pyTrue, pyFalse, pyEllipsis } from "../ast/constants.ts";
 import { pegen } from "./pegen_proxy.ts";
 import { Colors } from "../../deps.ts";
-import type { StartRule, CmpopExprPair, KeyValuePair, KeywordOrStarred, KeywordToken } from "./pegen_types.ts";
+import type { StartRule, CmpopExprPair, KeyValuePair, KeywordOrStarred } from "./pegen_types.ts";
+import { KeywordToken } from "./pegen_types.ts";
```

Comment:
I think the problem was that I was importing it as a type before because it wasn't being used previously

Reply:
thanks!